### PR TITLE
Added homepage

### PR DESCRIPTION
**Pull request by shegy28**
### Added Homepage for the portfolio website

**In this branch :**

I added and index file with the de
Added a CSS file used in styling the portfolio website
Added the images folder
Also updated the README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+            <header class="head-container">
+                <div class="logo">
+                    <h3>Asouzu Chimezie</h3>
```
Comment (aimalamiri) [inline index.html]: 1. It looks great, but it's highly recommended to put your logo in an anchor tag, so please ensure your logo is a link.
2. It's required to follow the exact Figma design, so please try using the same color text color for your logo.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                </div>
+                <div class="menu">
+                    <img src="/images/Union.png" alt="hamburger icon">
+                </div>
+            </header>
+        <section>
+            <div class="main">
+                <div class="wrapper">
+                    <div class="welcome-note">
+                        <h1>I'm Chimezie Glad to see you!</h1>
```
Comment (aimalamiri) [inline index.html:27]: It's fantastic 👌and you can make it look the same as Figma if you add some space to the top and use the same color for the title text. Kindly bring down the content, and update its color to match the original design.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                    <div class="socials">
+                        <p>LET'S CONNECT</p>
+                    </div>
+                    <div>
+                        <button><img src="/images/Vector.svg" alt=""></button>
+                        <button><img src="/images/Vector (1).svg" alt=""></button>
+                        <button><img src="/images/Vector (2).svg" alt=""></button>
+                        <button><img src="/images/Vector (3).svg" alt=""></button>
+                        <button><img src="/images/Linkedin icon.svg" alt=""></button>
+                    </div>
```
Comment (aimalamiri) [inline index.html]: Here you **listed** your social media accounts, right? so don't you think it's better to use `ul` and `li` tags for the **list** of your social media links?

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @shegy28, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The header section is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (shegy28) [reply index.html:27]: Thanks for the feedback i have done the necessary adjustments.

File: index.html (hunk 3 above)
Comment (shegy28) [reply index.html]: Thanks for the feedback i have done the necessary adjustments.

**Review by mwanawabangona — APPROVED** [review]
Hi @shegy28

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job Implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**